Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM alpine:3.3
 MAINTAINER Nicolas Degory <ndegory@axway.com>
 
+#Need to keep curl and bash after installation for containerPilot
 RUN apk update && \
-    apk --no-cache add python ca-certificates && \
-    apk --virtual envtpl-deps add --update py-pip python-dev curl && \
+    apk --no-cache add python ca-certificates curl bash && \
```

Comment:
LGTM  curl was permanently added later in the Dockerfile, but we can keep it here with bash if it's a prerequisite for all images. I just want to make sure we have the same layer defined here between all TICK images